examples/storage/cassandra/README.md

```diff
@@ -380,10 +380,10 @@ spec:
   replicas: 4
 ```
 
-The Pet Set will now contain four Pets.
+The StatefulSet will now contain four pods.
```

Comment:
I'm not sure if I understand this. Aren't those "pets" also pods? StatefulSet's status.replicas is calculated from the number of pods the controller sees, and the type of pets in the code is `api.Pod`